docker-compose.yml

```diff
@@ -12,6 +12,20 @@ services:
       - mysql:/var/lib/mysql
     healthcheck:
       test: ["CMD", "mysqladmin", "ping", "--silent"]
+  nginx:
+    build:
+      context: .
+      dockerfile: nginx.Dockerfile
+    restart: unless-stopped
+    environment:
+      HOSTNAME: 'changeme.example.com'
+      HTTP_PORT: 8080
+      YOTTER_ADDRESS: 'http://yotter:5000'
+      YTPROXY_ADDRESS: 'http://unix:/var/run/ytproxy/http-proxy.sock'
+    ports:
+      - "127.0.0.1:8080:8080"
+    volumes:
+      - "/var/run/ytproxy:/app/socket/"
```

Comment:
The whole point of having it as a socket is to let it bypass the docker proxy, to lower CPU usage. I'm afraid that using TCP for IPC communication isn't the best idea.

For small instances, however, I don't see a problem.

Reply:
You're taking a UNIX socket and converting it to a TCP socket. You lose more than 50% of your speed because of that.

https://blog.myhro.info/2017/01/how-fast-are-unix-domain-sockets

Reply:
Right - do you have a preferred solution in mind already?

FWIW I guess there should also be some adjustments to the docs here..?
https://github.com/ytorg/Yotter/blob/dev-indep/SELF-HOSTING.md#2-setting-up-yotter
https://paste.ubuntu.com/p/248hh6crWH/

Reply:
Yes, have the ytproxy independent of this nginx instance. It's just a waste of performance imo. 

What would you like to change in the documentation?

Reply:
Given that the result of following the linked guide is the same wrt this as the compose file here, if it's not ideal both of them should be changed, no?